Makefile_test.common

```diff
@@ -0,0 +1,37 @@
+define test_tpl =
```

Comment:
I would just rename it and call it a "common test" but rather a "common json diff test". Some plugin may use different testing strategies.

Reply:
Removed